app/src/substrate/substrate_coin.h

```diff
@@ -21,13 +21,14 @@ extern "C" {
 
 #define COIN_ADDR_TYPE 42
 #define CLA 0xbc
+#define CLA_RECOVERY 0x90
```

Comment:
Is this CLA agreed with the team? I don't remember seen it before. I may be wrong, just double checking

Reply:
I see now this is the polkadot CLA.... I am not so sure about this... it is not really the polkadot generic app...